icu4c/source/common/rbbidata.cpp

```diff
@@ -119,6 +120,9 @@ void RBBIDataWrapper::init(const RBBIDataHeader *data, UErrorCode &status) {
         return;
     }
 
+    UCPTrieValueWidth width = ucptrie_getValueWidth(fTrie);
+    U_ASSERT(width == UCPTRIE_VALUE_BITS_8 || width == UCPTRIE_VALUE_BITS_16);
```

Comment:
Not just an assert -- that goes away when not in debug mode. Please check explicitly and set an invalid-format-error if it's an unexpected value.

Reply:
done